index.js

```diff
@@ -3,92 +3,88 @@
 module.exports = {
   name: require('./package').name,
 
-  options: {
-    'ember-cli-babel': {
-      useBabelConfig: true
-    },
-    nodeAssets: {
-      ceibo: {
-        vendor: ['index.js']
-      },
-      jquery: {
-        vendor: ['dist/jquery.js'],
-        destDir: 'ecpo-jquery'
-      }
-    }
-  },
+  // options: {
+  //   'ember-cli-babel': {
+  //     useBabelConfig: true,
+  //   },
+  //   nodeAssets: {
+  //     ceibo: {
+  //       vendor: ['index.js'],
+  //     },
+  //     jquery: {
+  //       vendor: ['dist/jquery.js'],
+  //       destDir: 'ecpo-jquery',
+  //     },
+  //   },
+  // },
 
-  included() {
-    this.app = this._findHost();
+  // included() {
+  //   this.app = this._findHost();
 
-    this.import('vendor/ceibo/index.js', { type: 'test' });
+  //   this.import('vendor/ceibo/index.js', { type: 'test' });
 
-    this.import('vendor/ecpo-jquery/dist/jquery.js', { type: 'test' });
-    this.import('vendor/shims/ecpo-jquery.js', { type: 'test' });
+  //   this.import('vendor/ecpo-jquery/dist/jquery.js', { type: 'test' });
```

Comment:
going to extract it to an independent PR, which
 - removes the moduleFor adapters
 - removes ember-native-dom-helpers
 - removes custom jquery tricks. It was needed for the moduleFor adapters, which needed to use the same instance of jquery as the APP does, to re-use the same events methods. Now when the moduleFor are about to be removed the only remaining jquery responsibility is search. So it doesn't matter that big anymore which instance of jquery is used.